docs/rules/no-missing-link-fragments.md

```diff
@@ -1,68 +1,83 @@
 # no-missing-link-fragments
 
-Disallow link fragments that don't exist in the document.
+Ensures that link fragments (URLs that start with `#`) reference valid headings or anchors in the document. This rule helps prevent broken internal links.
```

Comment:
For our documentation, this should be a single sentence. I think the original was fine.

```suggestion
Disallow link fragments that don't exist in the document.
```